Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Build Stage
```

Comment:
Please note that corpus has been renamed to testsuite. Also, you don't need to run git clone here.